src/components/RepoPreview.js

```diff
@@ -1,8 +1,9 @@
 import React, {PropTypes} from 'react';
 import {Link} from 'react-router-dom';
+import Button from 'material-ui/Button';
 
 const RepoPreview = (props) => (
-    <div><Link to={`/${props.id}`}>{props.title}</Link></div>
+    <div><Button color="primary"><Link to={`/${props.id}`}>{props.title}</Link></Button></div>
```

Comment:
I do not think this is how `Button` was intended on being used; in that you have a `Link` in a `Button`. That feels weird to me and seems to be a shortcut to get the Link's functionality via a button. What is the intention here? Is it to navigate to a route? If so a `Link` is probably the most appropriate.

However, if you want to use a `Button` maybe it needs an `onClick` that then navigates to the proper route. You will need access to react-router's API to do this; something that I think is available in React's context. Since we have not covered context yet, maybe a `Link` is better for now and we can take a look at this more closely during our next pairing session.